.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: build
+
+on:
+  push:
+
+jobs:
+  build-linux-binary:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: gcc install
+        run: sudo apt install gcc-aarch64-linux-gnu
+      - name: Get tag
+        uses: little-core-labs/get-git-tag@v3.0.2
+        id: tag
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
any reason why we use 1.18 and not 1.17 like in the darwin and windows?

Reply:
I forgot to update in darwin and windows I will fix it